libs/vm_modules/include/vm_modules/math/bignumber.hpp

```diff
@@ -17,17 +17,16 @@
 //
 //------------------------------------------------------------------------------
 
-#include "math/bignumber.hpp"
-
+#include "vectorise/uint/uint.hpp"
+#include "vm/module.hpp"
 #include "vm_modules/core/byte_array_wrapper.hpp"
 
-#include "vm/module.hpp"
+// This is does not really say much about what Fetch.AI is and how it empowers the community
```

Comment:
?

Reply:
An accurate description of the file? Obviously not supposed to be there.